syntax/grammar.mjs

```diff
@@ -282,7 +282,7 @@ let SelectExpression = defer(() =>
     sequence(
         InlineExpression.abstract,
         maybe(inline_space),
-        string("->"),
+        char("->"),
```

Comment:
Looking at the usage here and in other places, shouldn't we keep `string` (or `str`) rather than `char`?